install/install.sh

```diff
@@ -24,11 +26,15 @@ fi
 
 
 BINARY="pixi-${ARCH}-${PLATFORM}"
+EXTENSION="tar.gz"
+if [[ $(uname -o) == "Msys" ]]; then
+  EXTENSION="zip"
+fi
 
 if [[ $VERSION == "latest" ]]; then
-  DOWNLOAD_URL=https://github.com/${REPO}/releases/latest/download/${BINARY}.tar.gz
+  DOWNLOAD_URL=https://github.com/${REPO}/releases/latest/download/${BINARY}.${EXTENSION}
 else
-  DOWNLOAD_URL=https://github.com/${REPO}/releases/download/${VERSION}/${BINARY}.tar.gz
+  DOWNLOAD_URL=https://github.com/${REPO}/releases/download/${VERSION}/.${EXTENSION}
```

Comment:
you remoevd `${BINARY}` here?

Reply:
Good catch! I just pushed the fix. I also confirmed locally that I can install a specific version:

```sh
PIXI_VERSION=v0.15.2 PIXI_HOME=./pixi-test bash install/install.sh
## This script will automatically download and install Pixi (v0.15.2) for you.
## Getting it from this url: https://github.com/prefix-dev/pixi/releases/download/v0.15.2/pixi-x86_64-pc-windows-msvc.zip
## ################################################################################################################ 100.0%
## Archive:  /tmp/.pixi_install.1FxREbAp
##   inflating: ./pixi-test/bin/pixi.exe
## The 'pixi' binary is installed into './pixi-test/bin'

./pixi-test/bin/pixi.exe --version
## pixi 0.15.2
```